### PR TITLE
Add Meshery Extensions Issue_Template

### LINKED ISSUE #7742: Adding Meshery Extensions Issue Templates in Meshery
#### Current Behavior

There's an absence of Meshery Extensions specific _ISSUE_TEMPLATE_ so there's a barrier in reporting bugs pertaining to Meshery Extensions. 

#### Desired Behavior

There's an _ISSUE_TEMPLATE_ for Meshery Extensions Bugs And Feature Requests.

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by 8rxn**
**Notes for Reviewers**

This PR fixes #7742 
Added new _ISSUE_TEMPLATES_ for Meshery Extensions Bug reports and Feature Requests.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/meshery-extensions_feature.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+#### Mockups
+<!-- Any visual diagrams of the desired user interface. -->
+
+---
+
+#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
+- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
+- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
+- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
+- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
```
[the remark this answers is not in the document]
Comment (8rxn) [reply .github/ISSUE_TEMPLATE/meshery-extensions_feature.md:31]: I think We could keep the general links like [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release), [UI Design for Meshery Extensions](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) and maybe also include the [Extensibility: Providers page](https://docs.meshery.io/extensibility/providers) from meshery Docs . Also tried looking up some related YouTube videos but couldn't found anything relevant to be added

[the remark this answers is not in the document]
Comment (leecalcote) [reply .github/ISSUE_TEMPLATE/meshery-extensions_feature.md:31]: Sounds real good. Depending upon how many there are, one could be a link to the API docs.

[the remark this answers is not in the document]
Comment (8rxn) [reply .github/ISSUE_TEMPLATE/meshery-extensions_feature.md:31]: Seems great. I'll make the change to include them.

[the remark this answers is not in the document]
Comment (8rxn) [reply .github/ISSUE_TEMPLATE/meshery-extensions_feature.md]: Hey @leecalcote 
Made the change in the contributor reference.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**